docs/ocis/adr/0008-configuration.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "8. Configuration"
+date: 2021-05-03T15:00:00+01:00
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/adr
+geekdocFilePath: 0008-configuration.md
+---
+
+# 5. Configuration Management for oCIS
```

Comment:
pls remove the header 1, it is in the front matter title

Reply:
```suggestion
## 5. Configuration Management for oCIS
```